### ISSUE TITLE
I can not seem to put the card

### ISSUE THREAD

**Issue by shyricaus** [jonassvensson4/jsfour-atm#4]
![image](https://user-images.githubusercontent.com/28937245/46300401-0d9ee800-c57a-11e8-9e6a-20c6fd543ffa.png)

**Comment by shyricaus** [comment]
size is good, and the card that is not catching

**Comment by jonassvensson4** [comment]
Never had this issue and no one have had this issue but you. So the problem is on your end.

Any errors?
What's the name of the folder?

**Comment by shyricaus** [comment]
when I try to put the card nothing happens

**Comment by shyricaus** [comment]
just do not go, do not go

**Comment by shyricaus** [comment]
name of the folder  jsfour-atm

**Comment by jonassvensson4** [comment]
I can't help you if there are no errors

**Comment by shyricaus** [comment]
What happens is that when I try to put the card, nothing happens nor an action

**Comment by jonassvensson4** [comment]
Yeah I understand what your problem is but I can't help you if there are no errors. I just downloaded the resource from Github to see if I missed something but it works fine for me. 

It must be something wrong with either your server or your client.

**Comment by shyricaus** [comment]
but that only happens when I am in resolution 1336x768
